src/renderer/entities/wallet/ui/SelectableShard/SelectableShard.tsx

```diff
@@ -0,0 +1,39 @@
+import { cnTw } from '@renderer/shared/lib/utils';
+import { BodyText, Checkbox, HelpText, Icon, Identicon, InfoPopover } from '@renderer/shared/ui';
+import { Address, Explorer } from '@renderer/shared/core';
+import { useAddressInfo } from '../../lib/useAddressInfo';
+
+type Props = {
+  name: string;
+  className?: string;
+  address: Address;
+  checked: boolean;
+  semiChecked?: boolean;
```

Comment:
Should it really have `semiChecked` prop or it works incorrectly now?

<img width="774" alt="image" src="https://github.com/novasamatech/nova-spektr/assets/5014869/b37b527f-8941-44b6-91a7-ec2cc4ec5cbc">

Reply:
Is it even needed for `ChainAccount`?
As I understand it can only be `checked` or `unchecked`

Reply:
Got it, thanks